theme/base-2016/record.twig

```diff
@@ -1,5 +1,4 @@
 {% extends 'partials/_master.twig' %}
-{% use 'partials/_sub_fields.twig' %}
```

Comment:
Oh wait … it has the template name in the block call … nothing to see here :rofl: